app/jobs/notify_email_job.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# This job is responsible for sending the notification to the Gov.UK Notify service.
+# It runs as a retryable job to handle the case where the Gov.UK service is temporarily offline.
+class NotifyEmailJob < ApplicationJob
+  queue_as :notifications
+
+  def perform(notification_id:)
+    notification = Notification.emails.kept.includes(:subscription).find(notification_id)
+    return unless notification.subscription.enabled?
+
+    if ENV.fetch('GOVUK_NOTIFY_API_KEY', nil).blank?
+      Rails.logger.error('[NotifyEmailJob] please set the GOVUK_NOTIFY_API_KEY env variable')
+      return # no point retrying later
+    end
+
+    if ENV.fetch('GOVUK_NOTIFY_TEMPLATE_ID', nil).blank?
+      Rails.logger.error('[NotifyEmailJob] please set the GOVUK_NOTIFY_TEMPLATE_ID env variable')
+      return # no point retrying later
+    end
+
+    response = nil
+    begin
+      response = MoveMailer.notify(notification).deliver!
```

Comment:
why this over deliver_now? It's not clear what the response for this is - can it ever return the error checked below?
https://github.com/ministryofjustice/govuk_notify_rails doesn't mention the deliver! method

Reply:
it seems that `.deliver!` is deprecated
https://coderwall.com/p/8yjt0a/actionmailer-method-deliver-is-already-deprecated-use-this-instead

Reply:
thanks, I'll switch to `deliver_now!`